apps/location-service/.env.example

```
@@ -1,7 +1,9 @@
 # optional: PORT=3000
 # optional: HEALTH_PORT=3001
 
-GOOGLE_PLACES_API_KEY=
 SIGNATURE_PUBLIC_KEY=
 
-BTC_EXCHANGE_RATE_URL_TO_REDIRECT_TO=
+# Standalone geocoding Postgres (see packages/geocoding-db)
+GEOCODING_DB_URL=postgresql://localhost:5433/geocoding
+GEOCODING_DB_USER=postgres
+GEOCODING_DB_PASSWORD=root
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GEOCODING_DB_PASSWORD key should go before the GEOCODING_DB_URL key

(UnorderedKey)